src/ast.ml

```diff
@@ -707,7 +713,7 @@ and
   | Func of
       loc *
       func_kind *
-      string list *  (* type parameters *)
+      (string * ghostness) list *  (* type parameters *)

```

Comment:
Let's not add support for real type parameters on functions (including lemma functions) yet?